7Series/DWOS-9.2-Tests.ps1

```diff
@@ -0,0 +1,14 @@
+describe '7Series System 9.2' {

```

Comment:
Why putting the version name in the file?

I don't think it's specific to this version